### ISSUE TITLE
Step fails on Ubuntu 22 stack

### ISSUE THREAD

**Issue by huszzsolt** [Genymobile/bitrise-step-genymotion-saas-install-gmsaas-cli#4]
This step fails after upgrading the Bitrise stack from Ubuntu 20->22.

Apparently `gmsaas`  executable was installed, but it can't be found?

<img width="1251" alt="Screenshot 2024-06-19 at 14 13 14" src="https://github.com/Genymobile/bitrise-step-genymotion-saas-install-gmsaas-cli/assets/28455856/54ddbe6a-b0b0-4ba9-9e64-b758fbf76e39">

<img width="1245" alt="Screenshot 2024-06-19 at 14 25 07" src="https://github.com/Genymobile/bitrise-step-genymotion-saas-install-gmsaas-cli/assets/28455856/21f55f59-357f-4fdb-986e-dcf44e20f6c0">

I have just submitted a Bitrise support ticket on this too.

**Comment by thomascarpentier** [comment]
Hi @huszzsolt,

Thanks for reporting this issue. 

We are working on a fix. I will inform you when the new version will be available. 

Regards.

**Comment by huszzsolt** [comment]
Hi @thomascarpentier,

I see you have a fix, but this is not published to Bitrise yet?

![Screenshot 2024-07-08 at 16 56 35](https://github.com/Genymobile/bitrise-step-genymotion-saas-install-gmsaas-cli/assets/28455856/d7fec827-27cc-498c-86de-9eecf3b998af)


Thanks,
Zsolt

**Comment by thomascarpentier** [comment]
Hi @huszzsolt , 

We have submitted a fix to Bitrise and we are waiting for the deployment. 

If you want to use it before,  you should modify your bitrise.yml in order to call the tag 0.0.3. 

Regards. 
Thomas

**Comment by huszzsolt** [comment]
Thanks @thomascarpentier ,

Updated bitrise yml from:
```
    - genymotion-saas-install-gmsaas-cli@0:
```
to
```
    - genymotion-saas-install-gmsaas-cli@0.0.3:
```
But it errors with because 0.0.3. If this is not correct how should we update the yml file?
Thanks,
Zsolt

**Comment by thomascarpentier** [comment]
Hi @huszzsolt , 

You should replace the line 
```- genymotion-saas-install-gmsaas-cli@0:```
by : 
```- git::https://github.com/Genymobile/bitrise-step-genymotion-saas-install-gmsaas-cli.git@0.0.3:```

**Comment by thomascarpentier** [comment]
Hi @huszzsolt , 

The step has been released on bitrise. You can use it now .

**Comment by huszzsolt** [comment]
Thanks, we noticed the step is live now. It is not backwards compatible with Ubuntu 20, but when we upgraded 20->22 all worked fine. So if anyone still using Ubuntu 20, should set their version fixed at 0.0.1.
Thanks for the fix I'll close this down.
